ui/app/templates/clients/client/index.hbs

```diff
@@ -294,12 +294,29 @@
           </button>
         {{/if}}
       </div>
-      <SearchBox
-        @searchTerm={{mut this.searchTerm}}
-        @onChange={{action this.resetPagination}}
-        @placeholder="Search allocations..."
-        @class="is-inline pull-right"
-        @inputClass="is-compact" />
+      <div class="pull-right is-subsection">
+        <MultiSelectDropdown
+          data-test-allocation-status-facet
+          @label="Status"
+          @options={{this.optionsAllocationStatus}}
+          @selection={{this.selectionStatus}}
+          @onSelect={{action "setFacetQueryParam" "qpStatus"}}
+        />
+        <MultiSelectDropdown
+          data-test-allocation-task-group-facet
+          @label="Task Group"
+          @options={{this.optionsTaskGroups}}
+          @selection={{this.selectionTaskGroup}}
+          @onSelect={{action "setFacetQueryParam" "qpTaskGroup"}}
+        />
+        <SearchBox
+          @searchTerm={{mut this.searchTerm}}
+          @onChange={{action this.resetPagination}}
+          @placeholder="Search allocations..."
+          @inputClass="is-compact"
+          @class="is-padded"
+        />
+      </div>
     </div>
     <div class="boxed-section-body is-full-bleed">
       <ListPagination
```

Comment:
It would be good to have an empty result placeholder for this table if there are no results after filtering. Otherwise it looks weird.
<img width="1246" alt="image" src="https://user-images.githubusercontent.com/775380/143092622-7e551d30-0d7a-4ef8-9aa2-0e537cd8b0b1.png">

Reply:
I like the idea of exploring this and appreciate the feedback, but I’m also kinda wanting to just ship the design we had decided on w/ Julie…maybe we can revisit down the road and iterate on it a bit?